src/components/CuratorDashboard/PendingRequestManager/PendingRequestTabs/BountyORChildBountySection.tsx

```
@@ -0,0 +1,275 @@
+			return item;
+		});
+		setPendingReq(updatedReqs || []);
+	};
+
+	const handleAcceptCuratorReq = async (data: IPendingCuratorReq) => {
+		if (!api || !apiReady || !loginAddress) return;
+
+		handleUpdateReq(data?.index, { loading: true });
+
+		let tx;
+		if (!!data?.parentBountyIndex && !isNaN(data?.parentBountyIndex)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)